### PR TITLE
Fundamental screwdriver build configuration

### PR DESCRIPTION
**Pull request by FenrirUnbound**
Configuration enables `npm install` & `npm test`

### CODE REVIEW

File: screwdriver.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+jobs:
+  main:
+    image: node:6
+    steps:
+      - init: npm install
+      - test: npm test
+  publish:
+    image: node:6
+    steps:
+      publish: npm publish
```
Comment (tkyi) [inline screwdriver.yaml]: I think all steps need a dash
http://screwdriver.cd/getting-started/yaml/

**Comment by nkatzman** [comment]
👍

**Comment by tkyi** [comment]
is publish usable right now?

**Comment by FenrirUnbound** [comment]
> @tkyi is publish usable right now?

Nope. Removed it from the configuration so it's not confusing

**Comment by tkyi** [comment]
👍  LGTM
giphy-4

**Comment by d2lam** [comment]
👍

**Pull request merged**